cli/replicaset/roles.go

```diff
@@ -5,19 +5,44 @@ import (
 	"slices"
 )
 
-// ChangeRoleFunc is a function type for addition or removing a role.
-type ChangeRoleFunc func([]string, string) ([]string, error)
+// RoleAction is a type that describes an action that will
+// happen with Roles. It can be "add" or "remove".
+type RoleAction uint
```

Comment:
`RoleAction` has `uint` type, but in the doc above you wrote that `It can be "add" or "remove"` (string values). Can you specify which `RoleAction` value (0/1) corresponds to which action (or omit mentioning them)?

Reply:
Added explanation for each action.